src/devices/MAX31856/README.md

```diff
@@ -0,0 +1,51 @@
+﻿# MAX31856
+
+## Summary
```

Comment:
Markdown 101 tip: there is a new line before and after all ## titles (and any other title except the first # which has only 1 line after and must be unique)